plugin/ale.vim

```diff
@@ -186,6 +186,15 @@ let g:ale_deno_executable = get(g:, 'ale_deno_executable', 'deno')
 " If 1, enable a popup menu for commands.
 let g:ale_popup_menu_enabled = get(g:, 'ale_popup_menu_enabled', has('gui_running'))
 
+" If 1, disables ALE's built in error display. Instead, all errors are piped
+" to Neovim's diagnostics API.
+let g:ale_send_to_neovim_diagnostics = get(g:, 'ale_send_to_neovim_diagnostics', 0)
+
+if g:ale_send_to_neovim_diagnostics && !has('nvim-0.6')
+    " no-custom-checks
+    echoerr('Cannot set g:ale_send_to_neovim_diagnostics to 1 unless you are running Neovim 0.6+.')
```

Comment:
```suggestion
    echoerr('g:ale_send_to_neovim_diagnostics should not be set to 1 unless you are running Neovim 0.6+.')
```

I think this is easier to understand.

Reply:
I reworded to `Setting g:ale_use_diagnostics_api to 1 requires Neovim 0.6+.`